app/assets/javascripts/home.js

```diff
@@ -50,21 +43,6 @@ ready = function() {
         },{offset: '75%'});
     });
 
-
-
-
-
-
-
-    function recalc(){
-        // pages will be scaled to 100% height
-        $('.page').outerHeight($(window).height())
-    }
-    $(window).resize(function(){
-        recalc();
-    });
-    recalc();
-
```

Comment:
I'm getting a recalc undefined error - I think this is used one other place too?

<img width="572" alt="screenshot 2017-12-22 20 09 16" src="https://user-images.githubusercontent.com/1441807/34315812-0bdf9fa0-e754-11e7-882f-f99fa417b196.png">